apps/web/vibes/soul/sections/reviews/index.tsx

```diff
@@ -85,12 +128,12 @@ export function ReviewsEmptyState({
   )
 }
 
-export function ReviewsSkeleton() {
+export function ReviewsSkeleton({ reviewsLabel = 'Reviews' }: { reviewsLabel?: string }) {
```

Comment:
I think exporting the Skeleton is still a great idea for those use cases when users want to stream in the entire data.